src/components/Header/Header.tsx

```diff
@@ -8,15 +8,20 @@ import { MenuItem } from "../MenuItem/MenuItem";
 import { menuItemData } from "../../assets/data/MenuItemData";
 import classNames from "classnames";
 
-const Header = () => {
+type Props = {
+  permanentBgColor: boolean;
+  scrollValue: number;
+};
+
+const Header = ({ permanentBgColor, scrollValue }: Props) => {
```

Comment:
we should add some defaults to make these optional to add in.

also all arbitrary values should be assigned to a const


```suggestion
const BACKOUND_COLOR_SCROLL_HEIGHT = 200

const Header = ({ staticBackgroundColor = "rgba(0,0,0,0)" , scrollValue = BACKOUND_COLOR_SCROLL_HEIGHT }: Props) => {
```

Reply:
This was still wanting the props to be filled out...